### ISSUE TITLE
k8s-bot generates invalid links

### ISSUE THREAD

**Issue by pmorie** [kubernetes/kubernetes#17282]
Looks like the test results are available from Jenkins in `console-log.txt`, not `build-log.txt`

**Comment by pmorie** [comment]
@kubernetes/goog-testing

**Comment by zhengguoyong** [comment]
i think #17312 say the same thing.
:)

**Comment by ixdy** [comment]
Neither one is entirely correct right now, unfortunately.

`build-log.txt` is what is uploaded by the [Google Cloud Storage Jenkins plugin](https://wiki.jenkins-ci.org/display/JENKINS/Google+Cloud+Storage+Plugin), which is what we have been using since I initially set up the per-PR Jenkins.

`console-log.txt` is what is uploaded by the [custom script I wrote](https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/upload-to-gcs.sh). I'd intended to replace the plugin with this script, except that I recently realized this isn't compatible with the Jenkins slaves I've now set up - the console log is accessible to the script only when run on the Jenkins master. As a result, we'll need to use the plugin to upload the log.

However, I also encountered a bug recently where the Jenkins plugin doesn't believe the Jenkins VM has proper GCP permissions, and is thus refusing to allow setting it up in projects or upload properly. I've been trying to fix this, but have been running into inscrutable Java stacktraces.

tl;dr hopefully the links will start working correctly again at some point soon. I'll update here when I believe everything has been resolved.

**Comment by ixdy** [comment]
OK, plugin is still not fully fixed, but it should be uploading `build-log.txt` again. Sorry for leaving this broken over the weekend.

**Comment by pmorie** [comment]
@ixdy Cool -- should I close this out or will you do a follow-up soon?

**Comment by ixdy** [comment]
I'll follow up with a close when I fix the plugin properly.

**Comment by pmorie** [comment]
No longer an issue, closing for now.
